hw2/04-background.html

```diff
@@ -1,29 +1,37 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>04 - Background</title>
     <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+            rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
-  </head>
-  <body>
-    <main>
-      <h1>Exercise 04 - Background</h1>
-      <p>
+</head>
+<body class="pt-5 p-2">
+<main class="card col-md-5 col-sm-12 mx-auto px-3 pt-3">
+    <h1>Exercise 04 - Background</h1>
+    <p>
         The background color will change every few seconds (as indicated below).
         You can stop this behavior and you can also change how often the
         background color changes.
-      </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
-      </div>
-      <div>
-        <input />
-      </div>
-    </main>
-  </body>
+    </p>
+    <div class="mb-2">
+        <label for="intervalInput"
+        >Background change interval (seconds):
+        </label>
+        <input type="number" id="intervalInput" value="3"/>
+    </div>
+    <div class="mb-2">
+        <input
+                type="button"
+                id="actionButton"
+                class="btn btn-block text-white h6"
+                value="Stop"
+        />
+    </div>
+</main>
+</body>
```

Comment:
do you have prettier/eslint turned on?

Reply:
I do. My code editor has different prettier configs for HTML files.